packages/playground/src/components/execution/step-execution.tsx

```diff
@@ -64,9 +64,9 @@ export const StepExecution: React.FC<StageExecutionProps> = ({ step, stepIndex }
             <StepExecutionToolbar />
           </Layout.Horizontal>
           <TabsContent value={StepExecutionTab.LOG}>
-            <div className="border-t pt-4">
+            <ScrollArea className="h-[calc(100vh-23rem)] border-t pt-4">
```

Comment:
is there a way to avoid these hardcoded heights? these items should take 100% height of the container ideally.
and let the container use flex/grid to set the height.
not a blocker.. I'll work with @jessieyu777 to figure out the layout.